examples/angularjs_require/test/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "private": true,
+  "devDependencies": {
+    "grunt": "0.4.5",
+    "grunt-contrib-jasmine": "0.8.1",
+    "grunt-template-jasmine-requirejs": "0.2.0",
```

Comment:
That would be inconsistent with what we're doing at the moment. The default assumption should be that they do. Also, yes, I do trust the grunt maintainers lol :)